src/main/java/core/basesyntax/service/ActivityService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Activity;
+import java.util.List;
+
+public interface ActivityService {
+    void analyze(List<Activity> activities);
+}
```

Comment:
let's unite your `ActivityService`, `ReportService` and part of your `ReportDao` logic

create a single `FruitService` interface with methods to save the data to storage and to get them from storage in form of a report
```suggestion
public interface FruitService {
    void saveData(List<FruitRecord> activities);
    
    String getFruitReport();
}
```